config/editor_profiles/default/configurations/SourceLabels.yml

```diff
@@ -2895,15 +2895,6 @@ asg:
     es: Asignatario
     pt: "Cessionário [asg]"
     pl: Cesjonariusz
-asn:
-  label:
-    it: Altro nome
-    fr: Autre nom
-    de: Weiterer Name
-    en: Associated name
-    pt: "Nome associado [asn]"
-    es: Nombre asociado
-    pl: Powiązana osoba
```

Comment:
@jenniferward could you double check this? (The key is duplicated but with different values)

Reply:
I don't see where this is used in Muscat; I can't find  "Weiterer Name" on any of the 700 relator code lists.

Reply:
@xhero could you check if this is used at all? I think it would be best to remove from the translation list all the relator codes that are not used.

Reply:
I would suggest punting the cleanup of the relator codes until after we've transitioned to the consolidated translations. It will probably be easier then.